### ISSUE TITLE
Config check complains about missing value for defined/not_defined conditions

### ISSUE THREAD

**Issue by tribut** [inverse-inc/packetfence#1487]
For example, the following from `conf/vlan_filters.conf.example` results in an error:

```
[machine]
filter = node_info.machine_account
operator = defined
```

`pfcmd checkup` says:

```
 FATAL - Missing value attribute in machine vlan filter rule
```

**Comment by julsemaan** [comment]
The same requirements is applied to all operators.

For now I believe adding the empty value is not something that is that bad

Tagging it for enhancement

**Comment by jrouzierinverse** [comment]
Fixed by 364ef20e
